search-app/src/cmr/search/api/routes.clj

```diff
@@ -56,6 +57,9 @@
         ;; Add routes for tagging
         tags-api/tag-api-routes
 
+        ;; Add routes for assoication
+        association-api/association-api-routes
+
```

Comment:
Like here, it's confusing to add routes for association, then followed by "Add routes for variable association".